docs/source/tutorial/08-add-a-details-view.mdx

```diff
@@ -82,6 +82,8 @@ This time, display a progressBar while the query executes:
             binding.error.visibility = View.GONE
 
             val response = apolloClient.query(LaunchDetailsQuery(id = args.launchId))
+        }
+    }
```

Comment:
I moved these up so this code would compile while you're working on it - otherwise everything freaks out